src/components/DropDownMenu.svelte

```diff
@@ -0,0 +1,27 @@
+<script lang="ts">
+  export let label: string;
+  export let id: string;
+  export let options: Record<string, string>;
+  export let selected: string;
+  export let placeholder: string;
+  export let onChange = () => {};
+  export let onSelect = () => {};
+  export let required = false;
+</script>
+
+<style>
+    option[value=""][disabled] {
+      display: none;
+    }
+</style>
+
+<label class="section-title" for={id}>{label}</label>
```

Comment:
According to the figma, the font size for the label is 14px, defined by ".label". ".section-title" is 25px;

Reply:
Are you saying to use `class="label"`?

Reply:
<img width="434" alt="Screenshot 2024-01-17 at 9 38 32 AM" src="https://github.com/LibertyDSNP/provider-dashboard/assets/43625033/3fc0cf6c-d6f7-4484-9cfd-2b0f40ff3ebc">

Yes. In app.css, the class label is defined with the correct styling for this element (14px). The class section-title is used for titles that have size 25px font, so should be deleted here.